Sources/Foundation/FileManager+POSIX.swift

```diff
@@ -459,18 +459,35 @@ extension FileManager {
         This method replaces pathContentOfSymbolicLinkAtPath:
      */
     internal func _destinationOfSymbolicLink(atPath path: String) throws -> String {
+        let bufferSize = Int(PATH_MAX + 1)
+        let buffer = try [Int8](unsafeUninitializedCapacity: bufferSize) { buffer, initializedCount in
+            let len = try _fileSystemRepresentation(withPath: path) {
+                readlink($0, buffer.baseAddress, bufferSize)
+            }
+            if len < 0 {
```

Comment:
Minor nit: I think that this could be better as:

```suggestion
            guard len >= 0 else {
              throw _NSErrorWithErrno(errno, reading: true, path: path)
            }
```

Reply:
Done.